testcode/mltopo.cpp

```diff
@@ -0,0 +1,180 @@
+/*BHEADER**********************************************************************
+ *
+ * Copyright (c) 2018, Lawrence Livermore National Security, LLC.
+ * Produced at the Lawrence Livermore National Laboratory.
+ * LLNL-CODE-745247. All Rights reserved. See file COPYRIGHT for details.
+ *
+ * This file is part of smoothG. For more information and source code
+ * availability, see https://www.github.com/llnl/smoothG.
+ *
+ * smoothG is free software; you can redistribute it and/or modify it under the
+ * terms of the GNU Lesser General Public License (as published by the Free
+ * Software Foundation) version 2.1 dated February 1999.
+ *
+ ***********************************************************************EHEADER*/
+
+/**
+   @file finitevolume.cpp
+   @brief This is an example for upscaling a graph Laplacian coming from a finite
+   volume discretization of a simple reservior model in parallel.
+
+   A simple way to run the example:
+
+   mpirun -n 4 ./finitevolume
+*/
+
+#include <fstream>
+#include <sstream>
+#include <mpi.h>
+
+#include "mfem.hpp"
+#include "../examples/spe10.hpp"
```

Comment:
is anything from `spe10.hpp` used here?

Reply:
good catch, I used it before, but not now.